agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -93,14 +93,26 @@ while true; do
 		;;
 	esac
 done
+
+function setup_ara {
+	generate_inventory >> /tmp/inventory.$$
+	check_install_rpm pbench-ara-role
+	ansible-playbook -i ${INVENTORY} /opt/pbench-agent/ansible/ara/pbench-ara.yml &>$pbench_log
+}
 	
 if $check; then
-	if [ "$sysinfo" == "all" ] || [ "$sysinfo" == "default" ] || [ "$sysinfo" == "none" ]; then
+	if [ "$sysinfo" == "default" ] || [ "$sysinfo" == "none" ]; then
 		:
+	elif [ "$sysinfo" == "all" ]; then
+		# generate inventory with pbench controller and remotes, install and configure ara
+		setup_ara
```

Comment:
Won't `all` get re-written to include `ara` now?  If so, we don't need this special case (lines 106-108).  If not, that would seem to indicate a problem.